docs/release/versioning-policy.md

```diff
@@ -0,0 +1,23 @@
+# Kubeflow Pipelines Versioning Policy
+
+The Kubeflow Pipelines version format follows [the semantic versioning](https://semver.org/).
```

Comment:
Suggestion revision:

The Kubeflow Pipelines version format follows [semantic versioning](https://semver.org/).

or

The Kubeflow Pipelines version format follows the [semantic versioning specification](https://semver.org/).